Week2/homework/js-exercises/ex3.js

```diff
@@ -0,0 +1,34 @@
+/*Declare a variable that holds an object
+Give the object 3 properties
+'string', number,array of strings*/
+const mealRecipe = {
+  title: 'Pizza',
+  servings: 4,
+  ingredients: ['Paste', '4 Tomato', 'Olive', 'Pepperoni', 'too much Cheese'],
+};
+//wile loop for title :'string' property and log it
+const myMeal = mealRecipe['title'];
+while (myMeal === 'Pizza') {
+  console.log(`Meal name: ${myMeal} 🍕`);
+  break;
+}
+// do loop for servings:number property and log it
+const serv = mealRecipe['servings'];
+let counter = 1;
+do {
+  console.log(`Serves:🍽  ${serv}`);
+  counter++;
+} while (counter === 1);
```

Comment:
I'm not sure what you're trying to do here. Because you're not looping through the properties?

Reply:
i think iam looping nothing here, it don't need any loop also

Reply:
but i want to use loop to display it , loop 1 time, so i will try to fix the same loop to give me 1 time and when i change number 1 to any number it give it also